docs/advanced-features/workflows.md

```diff
@@ -200,12 +210,32 @@ quilt3.Package().push(
 # Package test/package@06b2815 pushed to s3://quilt-sergey-dev-metadata
 ```
 
-Also `default_workflow` can be set in the config to specify which workflow will be used
-if `workflow` parameter is not provided.
+A `default_workflow` value can also be set at the top-level in your
+`config.yml` file:
+
+```yaml
+version:
+  base: "1"
+  catalog: "1"
+default_workflow: "experiment"
+is_workflow_required: False
+workflows:
+  experiment:
+    name: Experiment
+    metadata_schema: experiment-universal
+schemas:
+  experiment-universal:
+    url: s3://quilt-sergey-dev-metadata/.quilt/workflows/schemas/experiment-universal.json
```

Comment:
Echoing the previous comment about generalizing the S3 bucket name

Reply:
This is now fixed throughout